client/src/App.js

```diff
@@ -86,7 +86,8 @@ function App() {
                         <select id="version" value={config.version} name="version" className="form-select" aria-label="Default select example"
                                 onChange={e => update({version: e.target.value})}>
                           <option value="v5">YoloV5</option>
-                          <option value="v6">YoloV6</option>
+                          <option value="v6">YoloV6 R1</option>
+                          <option value="v6r2">YoloV6 R2 & R3</option>
```

Comment:
Let's perhaps change this to:
YoloV6 (latest) and YoloV6 (legacy - R1).
And let's move legacy one to the bottom.

Reply:
So that YoloV6 (legacy - R1) would be the lowest option?

Reply:
Yes @HonzaCuhel